src/dmd/mars.d

```diff
@@ -1971,6 +1989,34 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                 params.useInline = true;
                 params.hdrStripPlainFunctions = false;
             }
+            else if (p[1] == 'i')
+            {
+                params.includeImports = true;
+                if (p[2] != '\0')
+                {
+                    size_t start = 2 + (p[2] == '=');
+                    for (size_t nameIndex = start; ;nameIndex++)
+                    {
+                        if (p[nameIndex] == ',' || p[nameIndex] == '\0')
+                        {
+                            auto pattern = p + start;
+                            // TODO: we could check that the pattern only contains valid package characters
+                            //       if they aren't valid it's not a problem but would give the user a nice error message
+                            if (start == nameIndex)
+                            {
+                                error("invalid '%s', import package patterns cannot be empty", p);
```

Comment:
There doesn't appear to be any coverage on this.  Is it possible to add a test case?

Reply:
Also, should that read "'%s' is invalid, ..." instead?  Not sure.